app/assets/stylesheets/custom.css.scss

```diff
@@ -34,7 +33,7 @@ h1 {
    background-color: white;
    background: white;
    opacity: .8;
-   color: black;
+   color: #777;
```

Comment:
Color literals like `#777` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Line should be indented 2 spaces, but was indented 3 spaces